### PR TITLE
os/fs/smartfs: Update variable name in entry structure

### PR DESCRIPTION
**Pull request by amogh-samsung**
- Change variable name for file data length in entry structure
  from 'datlen' to 'datalen'.

Signed-off-by: Amogh Hassija <a.hassija@samsung.com>

### CODE REVIEW

**Review by thapav — COMMENTED** [review]
LGTM

File: os/fs/smartfs/smartfs.h [hunk 1]
```diff
@@ -267,7 +267,7 @@ struct smartfs_entry_s {
 	uint16_t flags;				/* Flags, including mode */
 	FAR char *name;				/* inode name */
 	uint32_t utc;				/* Time stamp */
-	uint32_t datlen;			/* Length of inode data */
+	uint32_t datalen;			/* Length of inode data */
 	uint16_t prev_parent;		/* To track the processed directory sector to find parent for entry. Holds last sctive sector in parent chain if new sector has tobe chained */
```
Comment (jeongchanKim) [inline os/fs/smartfs/smartfs.h:271]: Please modify sctive to active in description.

Comment (amogh-samsung) [reply os/fs/smartfs/smartfs.h]: Done

**Review by Taejun-Kwon — APPROVED** [review]

**Review by jeongchanKim — APPROVED** [review]
Looks good to me.

**Pull request merged**